.prettierignore

```diff
@@ -0,0 +1,62 @@
+dist
```

Comment:
I would personally prefer that we relied on eslint completely for prettier file selection and fixing, using [eslint overrides](https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns) rather than a massive .prettierignore file, and the `lintroller` task to apply fixes.